pytest

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec python -bb -m pytest "$@"
```

Comment:
Do we need this?  It's to make local testing easier but it's only and extra few chars.  Anyway, if we do then someone not on Windows will need to change the file executable flag 😉

Reply:
I vote we skip it, because it's confusing to have it in our repos like this.

Reply:
dropped in https://github.com/mozilla/addons-server/pull/10622/commits/5030b022401034a7110a2d41737ab90b52980b69